library/src/main/scala/giter8/JGitIgnore.scala

```diff
@@ -64,22 +64,39 @@ case class JGitIgnore(patterns: String*) {
 }
 
 object JGitIgnore {
+  private implicit class RichLine(val line: String) extends AnyVal {
+    def isCommentLine: Boolean = (line.nonEmpty && line(0) == '#') || line.trim.isEmpty
+  }
 
   def apply(in: InputStream): JGitIgnore = {
-    val patterns = Source.fromInputStream(in).getLines().toIndexedSeq
+    val source   = Source.fromInputStream(in)
+    val patterns = source.getLines().filterNot(_.isCommentLine).toSeq
+    // close here will cause scripted tests to fail
+    //source.close()
```

Comment:
```suggestion
    val source   = Source.fromInputStream(in)
    val patterns = try {
      source.getLines().filterNot(_.isCommentLine).toList
    } finally {
      source.close()
    }
```

avoid `toSeq` because `Iterator#toSeq` return `scala.Stream`